src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/StacksTests.cs

```diff
@@ -117,48 +119,10 @@ public Task TestJsonStacks(Architecture targetArchitecture)
                     Assert.NotNull(holder);
 
                     WebApi.Models.CallStackResult result = await JsonSerializer.DeserializeAsync<WebApi.Models.CallStackResult>(holder.Stream);
-
-                    var actualFrames = new List<WebApi.Models.CallStackFrame>();
-                    WebApi.Models.CallStackFrame[] expectedFrames = new WebApi.Models.CallStackFrame[]
-                    {
-                        new WebApi.Models.CallStackFrame
-                        {
-                            ModuleName = ExpectedModule,
-                            ClassName = ExpectedClass,
-                            MethodName = ExpectedCallbackFunction
-                        },
-                        new WebApi.Models.CallStackFrame
-                        {
-                            ModuleName = NativeFrame,
-                            ClassName = NativeFrame,
-                            MethodName = NativeFrame
-                        },
-                        new WebApi.Models.CallStackFrame
-                        {
-                            ModuleName = ExpectedModule,
-                            ClassName = ExpectedClass,
-                            MethodName = ExpectedFunction
-                        }
-                    };
-
-                    foreach (WebApi.Models.CallStack stack in result.Stacks)
-                    {
-                        actualFrames.Clear();
```

Comment:
@kkeirstead Your previous comment here turned out to be correct. I was only using the last stack's frames.